ci-operator/config/konveyor/pelorus/konveyor-pelorus-master.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  cli:
+    name: "4.9"
+    namespace: ocp
+    tag: cli
+  ubi_minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make dev-env
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.access.redhat.com/ubi8
+      USER root
+      RUN yum install -y python39 make diffutils openssl git && \
+          yum clean all && rm -rf /var/cache/dnf/*
+      RUN git clone https://github.com/konveyor/pelorus.git
+      ENV XDG_DATA_HOME=/.local
+      WORKDIR pelorus
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.9"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 500m
+      memory: 1Gi
+tests:
+- as: unit
+  commands: |
+    make lint
+  container:
+    from: src
+- as: e2e-openshift
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FIPS_ENABLED: "false"
+    test:
+    - as: e2e
+      commands: |
+        # TODO: optimize this
+        set -x
+        make dev-env
+        source .venv/bin/activate
+        export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+        oc create namespace pelorus
+        helm install operators charts/operators --namespace pelorus
```

Comment:
add "--debug" at the end of this line and line 55 so we know a bit more what's happening?